### DISCUSSION TITLE
Namuwiki에서 Adguard에 대해 극단적일 정도로 방해를 가하고 있는 것 같습니다.

### DISCUSSION THREAD

**Discussion by LDY0225** [List-KR/community#62] [AdGuard]
1)현재 Namuwiki의 스킨 espejo 스킨은 대단히 불편한 스킨임에도 불구하고 Adguard의 차단기능을 방해하고 있습니다.
2)차단한 광고, 사이드바, 오늘의 뉴스, 최근변경을 차단했음에도 불구하고 차단을 무시하거나
3)또는 광고를 차단한 상태에서 화면을 로딩하지 않거나 recaptcha를 지연시키고 있습니다.
4)또 사이드바를 차단하는 사용자 제작 필터를 입력하였을 경우 wiki 내용을 보여주지 않고 원래 차단해야 했을 사이드바만 남겨놓는 이상한 행위를 하는 것 같습니다. 아마 Namuwiki의 운영자 Umanle는 이런 식으로 Adguard 이용자를 Troll하거나 괴롭히기로 마음을 먹은 것으로 보입니다.
5)Liberty 스킨의 경우에는 본 필터인 List-KR 필터를 쓰면 Namuwiki의 내용 자체가 로딩되지 않습니다.
6)그리고 이 일련의 방해에 대해 Namuwiki 자체게시판의 관리자들은 거짓말 내지는 무시, 비판의 경우 차단으로 응대하고 있습니다.

부디 신속한 해결을 기원합니다.

**Comment by piquark6046** [comment]
Liberty 스킨을 사용하면 나무위키 콘텐츠를 제대로 로드할 수 없는 것으로 알고 있습니다.
신속한 해결은 어려울 것 같습니다. 쥐와 고양이 잡기 게임이 된 것 같네요.

**Reply by piquark6046** [reply]
근데 최신 NamuLink 유저스크립트로는 광고가 잘 차단되나요?

**Reply by hankju84** [reply]
광고 차단은 되는데 본문대로 일부 내용을 제대로 로딩하지 않는 현상이 발견됩니다. 그 상태에서 새로고침을 해야 해결되고요.

**Reply by piquark6046** [reply]
@hankju84 OS가 무엇인가요?

**Reply by hankju84** [reply]
윈도우와 iOS를 사용하고 있습니다. 특히 iOS 쪽에서 자주 일어나요.

**Reply by piquark6046** [reply]
어떤 문서를 읽으실 때 발생하나요?

**Reply by hankju84** [reply]
대문을 제외한 모든 문서에서 발생하는 듯합니다. 문단이 많을수록 발생 확률이 올라가는 것 같더군요.

**Reply by piquark6046** [reply]
@hankju84 재구현이 안됩니다.
https://drive.google.com/file/d/1EmtzTOIhy8IjCtpCmX0gTdIicobVu4VQ/view?usp=sharing

**Reply by piquark6046** [reply]
NamuLink 유저스크립트 버전이 3.16.3이 맞나요?
iOS 파일 관리자로 확인해주세요.

**Comment by hankju84** [comment]
3.16.3은 맞습니다.
아무래도 좀 더 실사용을 해봐야겠네요. 전 어제 한번 겪은지라...

**Reply by piquark6046** [reply]
재발할 경우에 https://github.com/List-KR/NamuLink/issues 에 새로운 이슈를 열어주세요.
